doc/ext/doctest.rst

```diff
@@ -55,13 +55,10 @@ names.
 
 .. rst:directive:: .. doctest:: [group]
 
-   A doctest-style code block.  You can use standard :mod:`doctest` flags for
-   controlling how actual output is compared with what you give as output.  By
-   default, these options are enabled: ``ELLIPSIS`` (allowing you to put
-   ellipses in the expected output that match anything in the actual output),
-   ``IGNORE_EXCEPTION_DETAIL`` (not comparing tracebacks),
-   ``DONT_ACCEPT_TRUE_FOR_1`` (by default, doctest accepts "True" in the output
-   where "1" is given -- this is a relic of pre-Python 2.2 times).
+   A doctest-style code block.  You can use standard :mod:`doctest`
```

Comment:
Here (and below) you rewrapped to something below 80 characters.  All other documents use 80 characters width.  Care to remain consistent?